### ISSUE TITLE
Feature: Don't remove what's already existing, update files only

### ISSUE THREAD

**Issue by cakehonolulu** [chabad360/hugo-gh-pages#23]
I'm in the need to use your workflow as it comes really handy, however I'd like to implement some sort of fail-safe like mechanism (Maybe configurable through the .yml) so that already existing files in the repo/branch that are missing in the hugo site don't get deleted.

Currently I use my site to host some bootcamp exercises (That I upload manually to the branch, as they're not SSG-ed) alongside my normal blog; but after each commit on the main branch, the action gets executed and I have to manually re-commit the entire bootcamp subdir to the 'deployment' branch.

Do you think it'd be doable?

Thanks for reading and congrats on the project!

**Comment by chabad360** [comment]
I do currently delete the destination folder on each run. Admittedly this may not be entirely necessary, but it seems like a good precaution to me.

One alternative option might be to just include the bootcamp content in the source folder (hugo should allow you to just put some straight content in a folder).

**Comment by cakehonolulu** [comment]
I agree; it's a good precaution as it doesn't leave residual/leftover files from the build; I 100% understand the decision behind this!

I've also thought about the alternative, I'll do that for now but I'll maybe fork the repo and try to add the feature myself, I've been learning slowly how GitHub actions works (Coming from Travis) and I'd love to take a deep dive on how Actions are coded.

Thanks for the input, greatly appreciate the answer!
